crates/edr_solidity_tests/tests/it/helpers/config.rs

```diff
@@ -130,7 +131,12 @@ impl<
     }
 
     /// Executes the test runner
-    pub async fn test(self) -> BTreeMap<String, SuiteResult<HaltReasonT>> {
+    pub async fn test(
+        self,
+    ) -> (
+        SolidityTestResult,
+        BTreeMap<String, SuiteResult<HaltReasonT>>,
+    ) {
```

Comment:
Suggestion(non-blocking): consider using a named struct to self-document the fields of the return type